maven-plugin/temp.json

```diff
@@ -0,0 +1,33 @@
+{"schema": {
+  "type": "object",
+  "properties": {
```

Comment:
Are you using this file somewhere? If yes, can you give a better name?

Reply:
I added this by mistake. I will remove it in the next commit.